libraries/third_party_libraries_manifest.json

```diff
@@ -311,11 +311,9 @@
   "zlib": {
     "product": "zlib",
     "vendor": "zlib",
-    "version": "1.3",
-    "commit": "09155eaa2f9270dc4ed1fa13e2b4b2613e6e4851",
-    "ignored-cves": [
-      "CVE-2023-45853"
```

Comment:
1.3.1 also fixes this CVE https://github.com/madler/zlib/issues/868#issuecomment-2655351405